JavaScript/JavaScriptSDK/Context/User.ts

```diff
@@ -29,12 +34,34 @@ module Microsoft.ApplicationInsights.Context {
          * The store region.
          */
         public storeRegion: string;
-        
+
+        static cookieSeparator: string = '|';
+
+        public setAuthId(id: string, accountId? : string) {
+            this.authId = id;
+            var authCookie = this.authId;
+
+            if (accountId) {
```

Comment:
just FYI if(0) will not pass - is it what you want (impossible to set accountid to 0)?